library/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="TappPageIndicator">
+        <!--
+                Will be the equivalent of LinearLayout.HORIZONTAL = 0 or LinearLayout.VERTICAL = 1
```

Comment:
I suggest here declaring our own constants for orientation, in order to not depend on another class